pkg/state/assets.go

```diff
@@ -160,7 +160,7 @@ func (a *assets) addNewRecord(assetID proto.AssetID, record *assetHistoryRecord,
 }
 
 func (a *assets) storeAssetInfo(assetID proto.AssetID, asset *assetInfo, blockID proto.BlockID) error {
-	assetConstBytes, err := asset.assetConstInfo.marshalBinary()
+	assetConstBytes, err := asset.marshalBinary()
```

Comment:
It's better to leave this without changes

Reply:
Resolved, see  https://github.com/wavesplatform/gowaves/pull/1644#discussion_r2017664220